kedro-datasets/kedro_datasets/pandas/deltatable_dataset.py

```diff
@@ -4,7 +4,7 @@
 """
 
 from copy import deepcopy
-from typing import Any, Optional
+from typing import Any, Optional, Union
```

Comment:
Same question about `Union` - shall we use `|` instead for consistency?

Reply:
Updated!